### ISSUE TITLE
use php-fpm

### ISSUE THREAD

**Issue by ghost** [indiehosters/remoteStorage#4]
Using php-fpm has a big performance improvement, consider using that instead of mod_php. See https://github.com/fkooman/php-remote-storage-deployment/blob/master/storage.example-httpd.conf for example Apache configuration using php-fpm.

**Comment by pierreozoux** [comment]
In this case, I'll use nginx in front.
(It is what I always do, but as your instructions were with apache..)

**Comment by ghost** [comment]
Yeah that is also good! I am not sure it was properly tested with nginx yet, so that would be helpful by itself! :D Note the X-Sendfile that was used and the limit on the request body.
